### DISCUSSION TITLE
Error codes

### DISCUSSION THREAD

**Discussion by wadclapp** [supabase/supabase#3873] [Questions] [answered]
```
.then(({ data, error }) => {
  console.log(error.code) // <- where can I find what the different codes mean?
})
```

Using abort and `.abortSignal(ac.signal)`, on fetching data and with rpc functions, I get `code: 20`, does that equate to aborted?

**Comment by silentworks** [comment]
Unfortunately these aren't documented, this is something the Supabase team is wary of and will address in the future.

**Reply by kylerummens** [reply]
Any update to this?

**Reply by steve-chavez** [reply]
@kylerummens The REST API error codes are documented now: https://postgrest.org/en/latest/errors.html

**Comment by steve-chavez** [comment, marked as answer]
> on fetching data and with rpc functions

If these are done through `postgrest-js`(or other client libs) then the code will come from PostgreSQL itself: https://www.postgresql.org/docs/current/errcodes-appendix.html

**Reply by steve-chavez** [reply]
More details at: https://postgrest.org/en/latest/errors.html
